docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,401 @@
+
+To see more information about inclusion proofs, see the 
+[staker/babylontypes.go](https://github.com/babylonlabs-io/btc-staker/blob/main/staker/babylontypes.go) file.
+
+## Proof of Possession
+
+Proof of Possession (PoP) is a signature that proves you own and control the 
+BTC private key used for staking. This proof is a required component when 
+creating a BTC delegation through `MsgCreateBTCDelegation`.
+
+To generate a Proof of Possession, you need to sign a specific message using 
+your BTC private key:
+
+```go
+pop, err := btcstaking.SignProofOfPossession(
+    privateKey,      // Your BTC private key
+    babylon_address, // Your Babylon address
+    net,            // Network parameters
+)
+```
+It is then returned as a `ProofOfPossessionBTC` message.
+
+The PoP serves as a security measure in the Babylon protocol, ensuring that only 
+the rightful owner of the BTC private key can create delegations. This 
+prevents unauthorized delegations and is a crucial part of the staking security 
+process.
+
+To see more information about Proof of Possession, see the 
+[staker/babylontypes.go](https://github.com/babylonlabs-io/btc-staker/blob/main/staker/babylontypes.go) file.
+
+The `MsgCreateBTCDelegation` message requires the Proof of Possession.
+
+## Other General Fields
+When creating a BTC delegation, several general fields are required in addition 
+to the signatures and proofs:
+
+**Staking Amount:**
+```go
+stakingValue := btcutil.Amount(1000000) // Amount in satoshis
+```
+
+**Version Parameters:**
+```go
+// Get current parameters version from Babylon chain
+params, err := babylonClient.GetBTCStakingParams(ctx)
+if err != nil {
+    return err
+}
+
+stakingTime := uint16(1000)    // Blocks for staking period
+unbondingTime := uint16(100)   // Blocks for unbonding period
+```
+
+**Fees:**
+```go   
+stakingFee := btcutil.Amount(1000)    // Fee for staking transaction
+slashingFee := btcutil.Amount(1000)   // Fee for slashing transaction
+unbondingFee := btcutil.Amount(1000)  // Fee for unbonding transaction
+```
+
+**Slashing rate:**
+```go
+slashingRate := float64(0.1) // 10% slashing rate
+```
+
+### Stake creation/registraton
+#### Registering a phase-1 stake
+
+To register your phase-1 stake to phase-2, you need to submit a 
+`MsgCreateBTCDelegation` with a valid inclusion proof. This requires your 
+staking transaction to already be confirmed on the Bitcoin network.
+
+1. Create `MsgCreateBTCDelegation` with the inclusion proof filled as defined 
+above and submit `MsgCreateBTCDelegation` to the Babylon network's btcstaking 
+module. If all verifications pass, your delegation becomes active on the Babylon 
+chain immediately since you've already proven your BTC is locked on Bitcoin.
+
+2. You can check the activation status by querying the babylon node like this.
+
+```bash
+babyloncli query btcstaking delegation-status <staking-tx-hash>
+```
+
+3. Note that certain eligibility criteria apply including:
```

Comment:
We don't need this, we should define the validity criteria on the `MsgCreateBTCDelegation` section.